Destiny.ScrimTracker.Api/Controllers/MatchController.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+using Destiny.ScrimTracker.Api.Requests;
+using Destiny.ScrimTracker.Logic.Models;
+using Destiny.ScrimTracker.Logic.Repositories;
+using Destiny.ScrimTracker.Logic.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Destiny.ScrimTracker.Api.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class MatchController : Controller
```

Comment:
@jaredonline also renamed this to be `Matches` in keeping with dem REST standards